app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -71,6 +72,7 @@ protected function create(array $data)
             'group_name' => $data['group_name'],
             'email' => $data['email'],
             'password' => Hash::make($data['password']),
+            // 'is_admin' => $data['is_admin']->nullable,
```

Comment:
Please do not make it a habit to push commented codes. You can push comments that explains a code but do not push commented codes.

Reply:
I understood.
I deleted the code.